packages/contracts-periphery/contracts/foundry-tests/citizen-house/BadgeAdmin.t.sol

```diff
@@ -0,0 +1,692 @@
+        address[] memory opcoAdrs = new address[](4);
+        if (set == 0) {
+            opcoAdrs[0] = mkadr("opco1");
+            opcoAdrs[1] = mkadr("opco2");
+            opcoAdrs[2] = mkadr("opco3");
+            opcoAdrs[3] = mkadr("opco4");
+        } else if (set == 1) {
+            opcoAdrs[0] = mkadr("opco5");
+            opcoAdrs[1] = mkadr("opco6");
+            opcoAdrs[2] = mkadr("opco7");
+            opcoAdrs[3] = mkadr("opco8");
+        } else if (set == 2) {
+            opcoAdrs[0] = mkadr("opco9");
+            opcoAdrs[1] = mkadr("opco10");
+            opcoAdrs[2] = mkadr("opco11");
+            opcoAdrs[3] = mkadr("opco12");
+        } else if (set == 3) {
+            opcoAdrs[0] = mkadr("opco13");
+            opcoAdrs[1] = mkadr("opco14");
+            opcoAdrs[2] = mkadr("opco15");
+            opcoAdrs[3] = mkadr("opco16");
+        } else if (set == 4) {
+            opcoAdrs[0] = mkadr("opco17");
+            opcoAdrs[1] = mkadr("opco18");
+            opcoAdrs[2] = mkadr("opco19");
+            opcoAdrs[3] = mkadr("opco20");
+        } else if (set == 5) {
+            opcoAdrs[0] = mkadr("opco21");
+            opcoAdrs[1] = mkadr("opco22");
+            opcoAdrs[2] = mkadr("opco23");
+            opcoAdrs[3] = mkadr("opco24");
+        }
+
+        return opcoAdrs;
+    }
+
+    function getCitizenSet(uint256 set) public pure returns (address[] memory) {
+        address[] memory citizenAdrs = new address[](4);
+        if (set == 0) {
+            citizenAdrs[0] = mkadr("citizen1");
+            citizenAdrs[1] = mkadr("citizen2");
+            citizenAdrs[2] = mkadr("citizen3");
+            citizenAdrs[3] = mkadr("citizen4");
+        } else if (set == 1) {
+            citizenAdrs[0] = mkadr("citizen5");
+            citizenAdrs[1] = mkadr("citizen6");
+            citizenAdrs[2] = mkadr("citizen7");
+            citizenAdrs[3] = mkadr("citizen8");
+        } else if (set == 2) {
+            citizenAdrs[0] = mkadr("citizen9");
+            citizenAdrs[1] = mkadr("citizen10");
+            citizenAdrs[2] = mkadr("citizen11");
+            citizenAdrs[3] = mkadr("citizen12");
+        } else if (set == 3) {
+            citizenAdrs[0] = mkadr("citizen13");
+            citizenAdrs[1] = mkadr("citizen14");
+            citizenAdrs[2] = mkadr("citizen15");
+            citizenAdrs[3] = mkadr("citizen16");
+        } else if (set == 4) {
+            citizenAdrs[0] = mkadr("citizen17");
+            citizenAdrs[1] = mkadr("citizen18");
+            citizenAdrs[2] = mkadr("citizen19");
+            citizenAdrs[3] = mkadr("citizen20");
+        } else if (set == 5) {
+            citizenAdrs[0] = mkadr("citizen21");
+            citizenAdrs[1] = mkadr("citizen22");
+            citizenAdrs[2] = mkadr("citizen23");
+            citizenAdrs[3] = mkadr("citizen24");
+        }
+
+        return citizenAdrs;
+    }
+
+    function getSupplySet(uint256 set) public pure returns (uint256[] memory) {
```

Comment:
Would prefer these to be constructed in the specific tests so that its easier to know what the values are in each test